Firebase/InstanceID/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# unreleased
+- [changed] Stop collecting logging ID as it is not used anymore.(#4732)
```

Comment:
Issue number is preferable to PR number here. (#4444)

Reply:
Done.